### ISSUE TITLE
关于XL训练模型报错

### ISSUE THREAD

**Issue by 175Roy** [aigc-apps/sd-webui-EasyPhoto#122]
Error executing the command: Command '['R:\\ai\\1\\sd-webui-aki\\sd-webui-aki-v4.4\\python\\python.exe', '-m', 'accelerate.commands.launch', '--mixed_precision=fp16', '--main_process_port=3456', 'R:\\ai\\1\\sd-webui-aki\\sd-webui-aki-v4.4\\extensions\\sd-webui-EasyPhoto\\scripts\\train_kohya/train_lora.py', '--pretrained_model_name_or_path=extensions\\sd-webui-EasyPhoto\\models\\stable-diffusion-v1-5', '--pretrained_model_ckpt=models\\Stable-diffusion\\SDXL 1.0 ArienMixXL 亚洲人像 亚洲人像_v2.0.safetensors', '--train_data_dir=outputs\\easyphoto-user-id-infos\\0924 asian 512\\processed_images', '--caption_column=text', '--resolution=512', '--random_flip', '--train_batch_size=1', '--gradient_accumulation_steps=1', '--dataloader_num_workers=0', '--max_train_steps=200', '--checkpointing_steps=200', '--learning_rate=0.0001', '--lr_scheduler=constant', '--lr_warmup_steps=0', '--train_text_encoder', '--seed=42', '--rank=128', '--network_alpha=64', '--validation_prompt=easyphoto_face, easyphoto, 1person', '--validation_steps=200', '--output_dir=outputs\\easyphoto-user-id-infos\\0924 sian 512\\user_weights', '--logging_dir=outputs\\easyphoto-user-id-infos\\0924 asian 512\\user_weights', '--enable_xformers_memory_efficient_attention', '--mixed_precision=fp16', '--template_dir=extensions\\sd-webui-EasyPhoto\\models\\training_templates', '--template_mask', '--merge_best_lora_based_face_id', '--merge_best_lora_name=0924  asian 512', '--cache_log_file=R:\\ai\\1\\sd-webui-aki\\sd-webui-aki-v4.4\\extensions\\sd-webui-EasyPhoto\\train_kohya_log.txt', '--validation']' returned non-zero exit status 1.

**Comment by wuziheng** [comment]
暂时不支持XL模型的训练，最近1周相关的更新会涉及XL的功能，尽请关注。
